tests/e2e/lib/flows/jetpack-connect.js

```diff
@@ -94,10 +94,10 @@ export async function syncJetpackPlanData( plan, mockPlanData = true ) {
 
 	const jetpackPage = await JetpackPage.visit( page, jetpackUrl );
 	await jetpackPage.openMyPlan();
-	await jetpackPage.reload( { waitFor: 'networkidle0' } );
+	await jetpackPage.reload( { waitFor: 'networkidle' } );
```

Comment:
It is 🤦 for me, but `waitFor` is the wrong option, and I'm afraid it never worked 🤦 🤦 🤦 


`waitUntil` should do the trick

Reply:
Indeed. Changed. Although, using `networkidle` was running into timeout most of the time, so I used `domcontentloaded`